### ISSUE TITLE
Bulk Import "Manual Activity Entry"

### ISSUE THREAD

**Issue by darrencope** [GoldenCheetah/GoldenCheetah#725]
It would be useful to have a way of importing a 'bulk' load of old data into GC to get started. With individual ride files from a Garmin, etc. this is very straightforward. However, many users (especially those who have just recently started training with power) have only a spreadsheet of summary data. with each row showing time, distance and other details for an individual ride, but the entire table showing many rides (potentially a whole season, or several seasons.) Despite lacking the detail of a full log, this data is still useful at a summary level (e.g. how many kms did I ride in 2008 vs. 2013.)

The parallel here is to the "Manual Activity Entry" tool, which already allows this to happen on a ride-by-ride basis. Essentially, a way to bulk-load this information is what I'm looking for.

TL;DR: Ideally there would be a way to bulk import a table (excel, csv, whatever) of summary data, and line up fields to match GC fields.

**Comment by amtriathlon** [comment]
Some time ago a user has posted a workaround for this in users forum: https://groups.google.com/d/msg/golden-cheetah-users/bNBpKusHDvs/dCowiJQoAgAJ
